dmoj/judgeenv.py

```diff
@@ -86,6 +86,8 @@ def load_env(cli=False, testsuite=False):  # pragma: no cover
                                  'security is left as an exercise for the reverse proxy)')
         parser.add_argument('-A', '--api-host', default='127.0.0.1',
                             help='IPv4 address to listen for judge API')
+        parser.add_argument('--polling-observer', action='store_true',
+                            help='use polling observer instead of observer')
```

Comment:
Maybe be more explicit, e.g. "use polling instead of inotify for problem updates", so that the reader doesn't need to know about the implementation details (`watchdog`'s `Observer` class) to use this flag.